config/application.rb

```diff
@@ -11,7 +13,13 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
-    # Settings in config/environments/* take precedence over those specified here.
+    config.time_zone = 'Asia/Yekaterinburg'
```

Comment:
```suggestion
    config.time_zone = 'Ekaterinburg'
```
список доступных таймзон можно посмотреть тут: https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html (в константе MAPPING)
твой вариант рабочий, но можно без указания региона